ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/model/IVIUpholdQuote.java

```diff
@@ -21,16 +22,29 @@
     private long expireInMillis;
 
     public static IVIUpholdQuote fromProto(UpholdQuote upholdQuote) {
-        return IVIUpholdQuote.builder()
+        var builder = IVIUpholdQuote.builder()
                 .quoteId(upholdQuote.getQuoteId())
-                .requestedAmount(new BigDecimal(upholdQuote.getRequestedAmount()))
                 .requestedCurrency(upholdQuote.getRequestedCurrency())
-                .quotedAmount(new BigDecimal(upholdQuote.getQuotedAmount()))
                 .quotedCurrency(upholdQuote.getQuotedCurrency())
-                .normalizedQuotedAmount(new BigDecimal(upholdQuote.getNormalizedQuotedAmount()))
-                .conversionFee(new BigDecimal(upholdQuote.getConversionFee()))
                 .createdAt(Instant.ofEpochMilli(upholdQuote.getCreatedAt()))
-                .expireInMillis(upholdQuote.getExpireInMillis())
-                .build();
+                .expireInMillis(upholdQuote.getExpireInMillis());
+
+        if (StringUtils.isNotBlank(upholdQuote.getRequestedAmount())) {
```

Comment:
Might need to make ProtoUtil an external lib :)

Reply:
agreed